content/sql/concepts/commands/terms/union/union.md

```diff
@@ -0,0 +1,59 @@
+---
+Title: 'UNION'
+Description: 'Combines the results of two or more SELECT queries.'
+Subjects:
+  - 'Data Science'
+  - 'Computer Science'
+Tags:
+  - 'Database'
+  - 'Queries'
+  - 'PostgreSQL'
+  - 'MySQL'
+  - 'SQLite'
+CatalogContent:
+  - 'learn-sql'
+  - 'paths/analyze-data-with-sql'
+  - 'paths/design-databases-with-postgresql'
```

Comment:
```suggestion
  - 'learn-sql'
  - 'paths/analyze-data-with-sql'
```

We should use two (2) links for the `CatalogContent` and fwiw the [paths/analyze-data-with-sql](https://www.codecademy.com/learn/paths/analyze-data-with-sql) link includes a section called "Joining Multiple Tables" and creating tables .